tools/src/main/java/org/apache/kafka/trogdor/workload/SustainedConnectionWorker.java

```diff
@@ -162,26 +163,24 @@ public void close() throws Exception {
             this.closeQuietly();
         }
 
+        protected void complete() {
```

Comment:
nit: `completeRefresh` to be more concrete in what it completes? Complete can be understood to mean a total completion of the connection, whereas this completes a single refresh (something finer-grained and more often)

Reply:
Done.